addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
```

Comment:
The problem is that with version `2.0` (and possibly `1.13`) the `didReceiveAttrs` hook gets called during the `_super` call. That would mean the `_processNewValue` gets called before `_sanitezedValue` has been set.